### ISSUE TITLE
nono303 2.4.5-2.4.7 apache won't load

### ISSUE THREAD

**Issue by Jon8RFC** [nono303/mod_md#6]
There is nothing being added to the apache logs.
Using nono303 mod_md 2.4.4, apache loads as expected.
Using the apache-included mod_md 2.4.5 it loads as expected.
Using nono303 mod_md 2.4.5, 2.4.6, 2.4.7 all will not load apache.

The file(s) most certainly are in the folder, but this is what EventViewer shows:
```
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Apache Service" /> 
  <EventID Qualifiers="0">3299</EventID> 
  <Level>2</Level> 
  <Task>0</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2021-09-28T23:28:19.000000000Z" /> 
  <EventRecordID>6613</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>Wilbur</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>The Apache service named</Data> 
  <Data /> 
  <Data>reported the following error: >>></Data> 
  <Data>httpd.exe: Syntax error on line 151 of C:/wamp64/bin/apache/apache2.4.49/conf/httpd.conf: Cannot load modules/mod_md.so into server: The specified module could not be found.</Data> 
  <Data /> 
  <Data /> 
  <Data /> 
  <Data /> 
  <Data /> 
  </EventData>
  </Event>
```

**Comment by nono303** [comment]
Hi @Jon8RFC 
As mentioned on the readme, mod_md >= 2.4.5 are now built with openssl 3.
I think your httpd & mod_ssl are running with openssl 1.1 and that explain the error (not the same dll required)
I'm actually looking to build back with openssl v1.1 as v3 is not provided on any common distribution.

**Comment by nono303** [comment]
Hi @Jon8RFC
mod_md >= 2.4.5 have been roll backed with openssl 1.1 build
the 3 tags are up to date. Can u have a test and give me a feedback ?
Thx!

**Comment by Jon8RFC** [comment]
All three work, thank you!

A couple of weeks ago, I had tried manually updating Apache with OpenSSL 3.0 files as I've done in the past with a similar handy resource like your files [here](https://slproweb.com/products/Win32OpenSSL.html), but it failed to start.  It looks like apache httpd still has an [open pull request](https://github.com/apache/httpd/pull/258) for necessary compatibility changes for 3.0.  The 1.1.1 changes were [implemented a few weeks after 1.1.1 release](https://github.com/apache/httpd/pull/53), so maybe it will be done soon.

Thanks for always compiling these!  I'm sure plenty of people, like me, silently use and appreciate the files.

**Comment by nono303** [comment]
Thx for your feedback @Jon8RFC !
Nothing to deal with mod_md but Fyi concerning OpenSSL 3.0 I have a "working" build for httpd / php / svn with a big dirty patch
[openssl3.patch.txt](https://github.com/nono303/mod_md/files/7253712/openssl3.patch.txt) related to https://github.com/nono303/win-build-scripts but runtime failed during cert renew with mod_md (see https://github.com/icing/mod_md/issues/263)
